stressor/stress-test.mts

```diff
@@ -6,9 +6,27 @@ import test, { run } from "node:test";
 import wrap from "word-wrap";
 import pLimit from "p-limit";
 import isEqual from "lodash.isequal";
+import { readFile, writeFile } from "node:fs/promises";
+import { Command, InvalidArgumentError } from "commander";
 
 const DEBUG = process.env.DEBUG === "true" || process.env.DEBUG === "1";
 const limitWorkflows = pLimit(8);
+function parseIntOption(value, dummyPrevious) {
```

Comment:
i think this typing should be:
```suggestion
function parseIntOption(value: string, dummyPrevious: number) {
```

Reply:
i like that idea!

Reply:
https://github.com/bocoup/aria-at-gh-actions-helper/pull/45